doc/README.md

```diff
@@ -81,4 +98,4 @@ License
 ---------------------
 Distributed under the [MIT software license](/COPYING).
 This product includes software developed by the OpenSSL Project for use in the [OpenSSL Toolkit](https://www.openssl.org/). This product includes
-cryptographic software written by Eric Young ([eay@cryptsoft.com](mailto:eay@cryptsoft.com)), and UPnP software written by Thomas Bernard.
+cryptographic software written by Eric Young ([eay@cryptsoft.com](mailto:eay@cryptsoft.com)), and UPnP software written by Thomas Bernard.
```

Comment:
What is changing here?

Reply:
It seems that the last '\n' has been deleted.

Reply:
Yes, not intentional but also not relevant (IMHO)...

Reply:
@fanquake I restored the last '\n' and now it seem like I added a blank line... LOL

Reply:
fixed